common/io/Serial.cpp

```diff
@@ -113,6 +115,7 @@ bool RemoveLockFile(const string &lock_file) {
   }
```

Comment:
And this function ^ wants renaming too please.

Reply:
Done - now it's RemoveUUCPLockFile.